hs/t/n/Err_InspectAbstractToken.rsh

```diff
@@ -0,0 +1,18 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
+  const A = Participant('A', {
+    params: Tuple(Token),
+  });
+  init();
+  A.only(() => {
+    const [ token ] = declassify(interact.params);
+    const mt = Maybe(Token).Some(token);
+  });
+  A.publish(mt);
+  commit();
+
+  A.pay([ maybe(mt, 0, (tok) => [ 5, tok ]) ]);
```

Comment:
What does this program do without your other changes? My assumption is that your code makes it so I can't even write
```
const x = fromMaybe(maybeTok, otherTok);
view.set(x);
```
because I would "inspecting" a token from a container. But the actual problem is that we aren't allowed to be paid in or pay with a token that we learned this way.